hack/buildLocal.sh

```
@@ -0,0 +1,6 @@
+#!/bin/bash
+eval $(crc oc-env) && \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)